Dockerfile

```diff
@@ -6,6 +6,11 @@ LABEL maintainer="BOSCoin Developers <devteam@boscoin.io>"
 COPY ./ /go/src/boscoin.io/sebak
 WORKDIR /go/src/boscoin.io/sebak
 
+# You probably don't need to change this
+ARG BUILD_MODE="install"
+ARG BUILD_ARGS=''
+ARG BUILD_PKG="./..."
```

Comment:
`ARG` is to specify variables used in the `Dockerfile`, while `ENV` is for environment variable in the container. Since those variables configure how `sebak` is build I felt they belong more to the former category than the later.